components/survey/Details/getQuestionsFromSurvey.js

```diff
@@ -0,0 +1,15 @@
+export const getQuestionsFromSurvey = survey => {
+  let questions = survey.data.relationships.questions.data;
+  const included = survey.included;
```

Comment:
```suggestion
  const includedObjects = survey.included;
```

or maybe `includedItems`. The main idea is to rapidly understands that this variable is an array of objects (and that these objects might not be only "questions" but can be something else). :)

Reply:
done in 0a12e6f2768d6cf6c93db4a407b806b7fc73a764